portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,70 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jonathan Ward's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 style="background-color:DodgerBlue;color:White;text-align:center">Jonathan Ward's Portfolio</h1>
+      <h2 style="text-align:center;">This is Jonathan's portfolio.</h2>
+
+      <p>I am rising junior at North Carolina Agricultural and Technical State University. I am majoring in Computer Engineering.</p>
+
+      <p>These are the links to my LinkedIn and Github:</p>
+
+      <a href="https://github.com/jeward414">Github</a>
+      <a href="http://linkedin.com/in/jonathan-ward4140">LinkedIn</a>
+
+      <h3 style="text-align:center;">About Me</h3>
+      <p>I was born in Raliegh, NC and moved all around when I was in elementary in school. In total I believe I went to at least 3 or 4 different elementary schools before we stoppped moving.
```

Comment:
Consider formatting your paragraphs so that it doesn't require scrolling when you're reading the source. It'll render the same way either way on the web, but it makes reading it a bit easier.